pentaho-js/src/main/javascript/cdf/components/ccc/BaseCccComponent.ext.js

```diff
@@ -40,7 +40,7 @@ define([
    * @type {string[]}
    * @private
    */
-  var __chartTypesBlackList = ['Bullet'];
+  var __chartTypesBlackList = ['Bullet', 'Funnel'];
```

Comment:
The CCC library has no funnel chart, and so this will not be used. I don't think anything is needed for this Viz. API visualization and CDF.

Reply:
Got it. Thanks for clarifying.